scripts/mppc_to_multiprobe.py

```diff
@@ -0,0 +1,339 @@
+                    px_size_scanimg * mppc_offset_px[1] / dist_per_V_diagCam[1])  # (x, y) [V]
+
+    return mppc_offset_px, mppc_offset_volt  # sign of value indicates direction of correction
+
+
+def main(args):
+    """
+    Handles the command line arguments.
+    :param args: (list) arguments passed.
+    :returns: (int) Value to return to the OS as program exit code.
+    """
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--debug', action="store_true", default=False)
+    parser.add_argument('--file', dest="file",
+                        help="Image file to run the mppc to multiprobe alignment for.")
+    parser.add_argument('--angle', dest="phi", required=True, type=float,
+                        help="Rotation angle in degrees to match the mppc image coordinate system "
+                             "with the coordinate system of the multiprobe. "
+                             "Can be automatically extracted with galvo_scan_to_multiprobe script "
+                             "(mirror angle in degrees).")
+    parser.add_argument('--gain', dest="gain_diagcam", required=True, nargs='+', type=float,
+                        help="The distance (gain) on the diagnostic camera per Volt applied on the AWGs"
+                             "in x and y direction. "
+                             "Can be automatically extracted with galvo_scan_to_multiprobe script "
+                             "(galvo gain in px/volt).")
+
+    parser.add_argument('--px_size_diagcam', dest="px_size_diagcam", type=int, default=75,
+                        help="The pixel size in nm on the diagnostic camera.")  # TODO depending on??
+    # TODO this needs to be an input arg or using the magnification to calc this pixel sizes!
+    parser.add_argument('--px_size_diagimg', dest="px_size_scanimg", type=int, default=40,
+                        help="The pixel size in nm on the scanned image. Default is 40nm assuming a magnification"
+                             "of 1000x. Please change if using a different magnification.")
+
+    options = parser.parse_args(args[1:])
+    if options.debug:
+        logging.getLogger().setLevel(logging.DEBUG)
+    else:
+        logging.getLogger().setLevel(logging.WARNING)
+
+    # If a file was specified, use this file. Otherwise, load last file from acquisition server.
+    if options.file:
+        image = tiff.read_data(options.file)[0]
+    else:
+        scan_path = '/home/sonic/asm_storage/'
+
+        scans = glob.glob(os.path.join(scan_path, "megafield_*"))
+        last_scan = max(scans, key=os.path.getctime)
+        image = TIFF.open(os.path.join(scan_path,
+                                       "{}/field_000_000_0_raw2.tiff".format(os.path.basename(last_scan)))).read_image()
+
+    try:
+        # pre-process image
+        corner_images, image_E4 = preprocess_images(image)
+
+        # calculate mppc offset, Note: accuracy on AWGs is 0.001 volt
+        mppc_offset_px, mppc_offset_volt = \
```

Comment:
Personally I think it looks better for mulit-line code not to have to use the "\". I would write this as:
```
mppc_offset_px, mppc_offset_volt = get_mppc_offset(
    image_E4, math.radians(options.phi), tuple(options.gain_diagcam), options.px_size_diagcam, options.px_size_scanimg)
```